homeassistant/components/voip/devices.py

```diff
@@ -92,9 +98,16 @@ def async_setup(self) -> None:
             )
             if voip_id is None:
                 continue
+            device_store = self.device_stores[voip_id] = Store(
+                self.hass, STORAGE_VER, f"voip-device-{voip_id}"
+            )
+            device_data: dict[str, Any] = await device_store.async_load() or {}
```

Comment:
I am wondering if it would make sense to create just 1 store for all voip related things.

@farmio as you use Store in KNX, can you maybe share experience and guide them in a direction for voip?